### DISCUSSION TITLE
Code_ESP32_Blynk_Bluetooth_WiFi_Switch_control_8_Relays_02.ino:27:21: error: variable or field 'button1Handler' declared void

### DISCUSSION THREAD

**Discussion by Bibekjrb123** [bxparks/AceButton#115] [General]
Any one here to solve it please

**Comment by bxparks** [comment]
It is impossible to answer this question without access to the code. Please provide the [minimum reproducible example](https://en.wikipedia.org/wiki/Minimal_reproducible_example).

**Reply by bxparks** [reply]
No response after 3 weeks.
Closing to remove clutter, since this provides no information to anyone else in the future.
